app/main.py

```diff
@@ -1,13 +1,27 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
+            format_linter_error(error=every) for every in errors
+            if every["filename"] == file_path
+        ],
```

Comment:
There is no need in the if condition, all errors will correspond to the `file_path` because you pass them to the function in this way. Simplify this code:
```suggestion
        "errors": [format_linter_error(error) for error in errors],
```